package.json

```diff
@@ -64,19 +65,20 @@
     "jest": "^23.6.0",
     "postcss-loader": "^3.0.0",
     "postcss-preset-env": "^6.5.0",
-    "react": "^16.7.0",
-    "react-dom": "^16.7.0",
+    "react": "^16.9.0",
+    "react-dom": "^16.9.0",
     "react-ga": "^2.5.6",
-    "react-styleguidist": "^8.0.6",
+    "react-styleguidist": "9.0.4",
```

Comment:
Interestingly, we cannot upgrade more then this for the time being because we do not yet support the React.Fragment shorthand and they started using this in a couple of places 😛